projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-jetpack-token-subscription-service.php

```diff
@@ -17,13 +17,23 @@
  */
 class Jetpack_Token_Subscription_Service extends Token_Subscription_Service {
 
+	/**
+	 * Is this code executing on WPCOM?
+	 *
+	 * @return bool True if executing on WPCOM.
+	 */
+	private function is_wpcom() {
```

Comment:
there's a helper both in Jetpack and WPCOM that does this check that we can use instead

https://github.com/Automattic/jetpack/blob/df6ffd8d211e5596d28e525787cb50a0803f609b/projects/packages/status/src/class-host.php#L66